turborepo-tests/integration/tests/ordered/stream.t

```diff
@@ -0,0 +1,28 @@
+# Setup
+  $ . ${TESTDIR}/../../../helpers/setup.sh
+  $ . ${TESTDIR}/../_helpers/setup_monorepo.sh $(pwd) ordered
+
+# Build in stream order. All the .*'s are unpredictable lines, however the amount of lines is predictable.
```

Comment:
The current test should adequately determine if the logs are grouped or streamed. Specifically the streamed test seems like it might not do a good job, but it actually does. A lot of the lines are completely unpredictable, however, the last 3 are always the same for `stream` and are always different than the 3 lines for `grouped`.

The lines that currently match the `.*` regex only depend on which task starts first, which isn't deterministic and not influenced by `log-order`, the last 3 lines are based on the timing of the commands, which is deterministic, and the `log-order`.

While I could make a script that verifies it in more detail, I do not see any benefit in doing so in this case.

**TL;DR** The current `stream.t` test will fail if grouped logs are passed to it, and it should adequately test if logs are grouped or not.

Reply:
that makes sense. can you add some comments to this effect into the test?